frontend/src/pages/Afterword/index.js

```diff
@@ -73,89 +73,78 @@ const Button = styled(CarbonButton)`
 `;
 
 type Props = {
-  afterwordMessage: string,
   match: any,
   history: *,
-  debrief: boolean,
-  scores: Array<any>,
 };
 
 const clearSessionStorage = async () => await sessionStorage.clear();
 
-class Afterword extends React.Component<Props, State> {
-  async componentDidMount() {
-    getRange(0, 100).map(i => clearInterval(i)); //not the best solution
-    clearSessionStorage().then(() => {
-      persistor.purge();
-    });
-    const { participantUuid } = this.props.match.params;
-    this.props.getDebrief(participantUuid);
-  }
-
-  getHeaders = () => [
+const Afterword = (props: Props) => {
+  const { match } = props;
+  const { participantUuid } = match.params;
+  const { debrief, scores } = useSelector(state => state.debrief);
+  const afterwordMessage = useSelector(state => state.exercise.afterword);
+  const dispatch = useDispatch();
+  const getHeaders = () => [
     { key: 'task', header: 'Task' },
     { key: 'score', header: 'Score' },
     { key: 'debrief', header: 'Message' },
   ];
 
-  render() {
-    const { afterwordMessage, match, debrief, scores } = this.props;
-    const { participantUuid } = match.params;
+  useEffect(() => {
+    async function didMount() {
+      getRange(0, 100).map(i => clearInterval(i)); //not the best solution
+      await clearSessionStorage();
+      persistor.purge();
```

Comment:
I think we should add an await here too as it returns a promise
```suggestion
      await persistor.purge();
```